### DISCUSSION TITLE
OS and system updates

### DISCUSSION THREAD

**Discussion by jjjeroenski** [micro-nova/AmpliPi#239] [Q&A]
How will you handle operating system and system updates in the future?
Is there a roadmap for coming releases?

**Comment by linknum23** [comment]
> How will you handle operating system and system updates in the future?

For the near future system/os updates added to the update process. Later on we will consider doing something similar to Home Assistant's Application/Supervisor container model to make easier and more foolproof updates.

> Is there a roadmap for coming releases?

We do not have a roadmap yet. Our plan is to keep our [project board](https://github.com/micro-nova/AmpliPi/projects/1) up to date and add "Planned" and  "Up Next" categories. At the moment it is a couple of months out of date, though.

That said our engineers are working on the following:
- Android app updates to add URL/IP entry and AmpliPi hostname checks https://github.com/micro-nova/Amplipi-Mobile-App/issues
- Robustifying the underlying embedded firmware #240 
- Integrating HA #184 and using Server Sent Events #73 to make the integration more responsive
- Audio Level Equalization #186 
- Bluetooth #150
